.github/workflows/nodejs.yml

```
@@ -0,0 +1,23 @@
+name: Node CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x]
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: yarn install, build, lint
+        run: |
+          yarn
```

```suggestion
          yarn --frozen-lockfile
```

> If you need reproducible dependencies, which is usually the case with the continuous integration systems, you should pass --frozen-lockfile flag.
https://classic.yarnpkg.com/lang/en/docs/cli/install/

---

thanks, how i forgot it 🙈